cli/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
We could insert `node_modules/` into the `.gitignore`  in the root folder, so isn't necessary to repeat it here.